homeassistant/components/recorder/websocket_api.py

```diff
@@ -221,7 +227,7 @@ async def ws_handle_get_statistics_during_period(
         vol.Required("type"): "recorder/statistics_during_period",
         vol.Required("start_time"): str,
         vol.Optional("end_time"): str,
-        vol.Optional("statistic_ids"): [str],
+        vol.Required("statistic_ids"): [str],
```

Comment:
let's validate at least 1 is passed in

Reply:
Removed and switched to vol.Length in the schema 8d3fea260d1067a9741560abf8687122a027a3de